cmd/gentypes/gentypes.go

```diff
@@ -379,6 +383,11 @@ type Message interface {
 
 `
 
+// typesBlacklist is a blacklist of type names we don't want to emit.
+var typesBlacklist = map[string]bool{
+	"LaunchRequestArguments": true,
```

Comment:
I think you either need a comment here or you need to add this item to the list where you have the special case for using map for the arguments type. Otherwise it is not clear what this is all about on its own.

Reply:
Added a comment. I can't move it to the other place because there it's about fields, not top-level types.

An alternative would be to drop the blacklist and just emit this type, even though it's not being used now. When considering it, I decided to go with the current approach because our design goal is to keep `schematypes.go` as readable as possible, without mysterious unused types.